src/wrap/WrapperMatrix1D.js

```diff
@@ -0,0 +1,31 @@
+import AbstractMatrix from '../abstractMatrix';
+
+export default class WrapperMatrix1D extends AbstractMatrix() {
+    constructor(data, options = {}) {
+        const {
+            rows = 1
```

Comment:
should we also accept a "columns" option? Or maybe keep that for another class that stores the data column-wise instead of row-wise?